src/db/sqlite/lrsql/sqlite/record.clj

```diff
@@ -103,6 +103,8 @@
       (migrate-timestamps-activity-profile-03! tx)
       (migrate-timestamps-activity-profile-04! tx))
     (update-schema-simple! tx alter-credential-to-scope-scope-datatype!)
+    (when-not (= "CASCADE" (:on-delete (first (query-statement-to-actor-has-cascade-delete? tx))))
```

Comment:
pertaining to above, it seems we do have the check here, so do similar for pg for sure.

Reply:
duplicated (equivalent) to postgres

Reply:
i am not seeing it in the latest commit

Reply:
the logic is in record.clj rather than the SQL; I couldn't figure out how to do a conditional ALTER TABLE in plain Postgres SQL

https://github.com/yetanalytics/lrsql/blob/ca19a7e20d1ca68765b53d13565ed943271b811d/src/db/postgres/lrsql/postgres/record.clj#L70